FirebaseAuth/Sources/Swift/MultiFactor/TOTP/TOTPSecret.swift

```diff
@@ -54,7 +54,7 @@ import Foundation
     ///
     /// See more details
     /// [here](https://developer.apple.com/documentation/authenticationservices/securing_logins_with_icloud_keychain_verification_codes)
-    @objc(openInOTPAppWithQRCodeURL:)
+    @MainActor @objc(openInOTPAppWithQRCodeURL:)
```

Comment:
Done in https://github.com/firebase/firebase-ios-sdk/pull/new/nc/auth
```suggestion
    @objc(openInOTPAppWithQRCodeURL:)
```